README.md

```diff
@@ -1,2 +1,28 @@
-# engineering-challenge
-Demonstrate your experience, skills, and creativity https://airspacelink.com/careers/
+## Description
+
+Thanks to your [Part 107 Commercial Drone Pilot's license](https://www.faa.gov/uas/), you've been hired by a local electrical company to inspect several of their power lines using aerial drone videography. To understand if you are legally allowed to fly in this area, you've requested data from the FAA to indicate what airspace, if any, near your flight is [controlled](https://www.faa.gov/uas/recreational_fliers/where_can_i_fly/airspace_101/) and thus not eligible for drone flights. The FAA returned you an array of coordinates in [geoJson](https://geojson.org/) representing a single polygon of controlled airspace. Your job is to create a web application that will allow you to sketch various possible flights to visualize and quantify which portions of your flight will be in controlled airspace, so you can adjust as necessary.
+
+## Requirements
+
+The tools you will need are listed in the Tools section below. At its most basic, your app should load a map centered over the FAA polygon, which is also visible. The user should have the ability to draw shapes on the map, and after the sketch is complete, it will be visualized with a different color than the one used for the FAA shape. You should also be able to clearly visualize which portion of your flight area is NOT in controlled airspace (in other words, the area that doesn't intersect). Finally, display a message on the page indicating whether this flight area will be approved or not.
+
+At Airspace Link, we feel strongly about good communication. Make sure to provide a `README.md` which explains your approach, both from a functional and code perspective. Pay attention to grammar; good writing will always win us over. The README should also give us instructions about how to get it up and running after the repo is cloned. You can also write up a list of great ideas you'd love to tackle on this app if you had infinite time to work on it.
```

Comment:
Both of the last 2 sentences use the word "also".  The last sentence could be changed to start: "In addition, you could write up a list..." or, even better, it could be moved entirely to the list of extra credit ideas since it is duplicated there anyway.

Reply:
Good suggestions, I'll incorporate them. Thanks.